packages/@ngtools/webpack/src/loader.ts

```diff
@@ -248,14 +225,72 @@ function _replaceBootstrap(plugin: AotPlugin, refactor: TypeScriptFileRefactor)
     // Add the appropriate import
     refactor.insertImport(platform.name, platform.importLocation);
   });
+}
+
+
+function _replaceBootstrap(refactor: TypeScriptFileRefactor) {
+  // If bootstrapModule can't be found, bail out early.
+  if (!refactor.sourceMatch(/\bbootstrapModule\b/)) {
+    return;
+  }
+
+  const bootstraps = refactor.findAstNodes(refactor.sourceFile, ts.SyntaxKind.CallExpression, true)
+    .filter((call: ts.CallExpression) =>
+      call.expression.kind == ts.SyntaxKind.PropertyAccessExpression)
+    .map((call: ts.CallExpression) => call.expression as ts.PropertyAccessExpression)
+    .filter(access => {
+      return access.name.kind == ts.SyntaxKind.Identifier
+        && access.name.text == 'bootstrapModule';
+    });
+
+  if (bootstraps.length == 0) {
+    // Didn't find any dynamic bootstrapping going on.
+    return;
+  }
+
+  _replacePlatform(refactor, bootstraps);
 
   bootstraps
     .forEach((bs: ts.PropertyAccessExpression) => {
       // This changes the call.
       refactor.replaceNode(bs.name, 'bootstrapModuleFactory');
     });
+}
+
+
+function _replaceEntryModule(plugin: AotPlugin, refactor: TypeScriptFileRefactor) {
+  const modules = refactor.findAstNodes(refactor.sourceFile, ts.SyntaxKind.Identifier, true)
```

Comment:
I made changes to the code and added tests to account for this. It recursively searches for a function call now.

The reasoning for this is that the engines over at https://github.com/angular/universal offer methods which take in a module and obfuscate the core apis into files which do not pass through the loader so this would broaden it to the point where all the engines would work.